src/main/java/eva/Eva.java

```diff
@@ -0,0 +1,65 @@
+package eva;
+
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import eva.exceptions.TaskException;
+import eva.storage.Handler;
+import eva.tasks.Task;
+import eva.ui.Ui;
+
+/**
+ * Represents the main class of the Eva program. Contains the main method to run the program.
+ * The Eva program is a task manager that allows users to add, delete, mark as done, and list tasks.
+ *
+ * <p> The Eva program is able to handle three types of tasks:
+ * <ul>
+ *     <li> Todo: A task without any date/time attached to it.</li>
+ *     <li> Deadline: A task that needs to be done before a specific date/time.</li>
+ *     <li> Event: A task that starts at a specific time and ends at a specific time.</li>
+ * </ul>
+ * </p>
+ *
+ * @author Farhan Navas
+ * @version 1.0
+ * @since 2025-09-01
+ */
+public class Eva {
+    public static ArrayList<Task> taskList  = new ArrayList<>();
```

Comment:
Perhaps you can remove the extra space before "="?
```suggestion
    public static ArrayList<Task> taskList = new ArrayList<>();
```

Reply:
> Perhaps you can remove the extra space before "="?

Same as what veehz has mentioned, could remove the extra whitespace to make the code cleaner